articles/how-to-build-astatic-site-with-gatsbyjs/index.md

```diff
@@ -0,0 +1,206 @@
+---
+layout: engineering-education
+status: publish
+published: true
+url: /engineering-education/history-of-nodejs/
+title: How to build a static site with GatsbyJs
+description: Node.js is a runtime server environment that uses JavaScript on the server side and asynchronous programming. It is a free and open source technology that runs on various platforms (Mac OS X, Unix, Windows, etc.)
+author: jethro-magaji
+date: 2020-08-25T00:00:00-08:00
+topics: [Node.js]
+excerpt_separator: <!--more-->
+images:
+
+  - url: /engineering-education/history-of-nodejs/hero.jpg
+    alt: hero example image computer Node.js
+---
+When I first heard about GatsbyJs I was pretty confused and related it to the famous ***‘J Gatsby’*** of the movie [“The Great Gatsby”](https://www.imdb.com/title/tt1343092/), my goal for this article is not to confuse you but to get you started in building your very first static site using GatsbyJs.
```

Comment:
I really enjoy this as it adds a level of personality to your article - but we need to find a way to include it and not confuse the audience or not make it seem likes its just a random fact out. 
Lets consider better placement and/or omission.

Reply:
Okay, I restructured the whole sentence.